x-pack/metricbeat/module/mssql/log/_meta/data.json

```diff
@@ -0,0 +1,54 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "agent": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
+    },
+    "event": {
+        "dataset": "mssql.log",
+        "duration": 115000,
+        "module": "mssql"
+    },
+    "metricset": {
+        "name": "log"
+    },
+    "mssql": {
+        "database": {
+            "id": 1,
+            "name": "master"
+        },
+        "log": {
+            "space_usage": {
+                "since_last_backup": {
+                    "bytes": 94208
+                },
+                "total": {
+                    "bytes": 2088960
+                },
+                "used": {
+                    "bytes": 688128,
+                    "pct": 32.94117736816406
+                }
+            },
+            "stats": {
+                "active_size": {
+                    "bytes": 94207.213568
+                },
+                "backup_time": "1900-01-01T00:00:00Z",
```

Comment:
Interesting timestamp :-)

Reply:
In fact, it's correct! It seems that it's the default value that MSSQL returns when no value is set (and it's not nil cell).

![image](https://user-images.githubusercontent.com/4249331/51319999-d2f48900-1a5e-11e9-8688-5a07281a2d8f.png)